playbooks/configure-python-sources.yml

```diff
@@ -118,6 +133,14 @@
         - py_artifact_found | bool
         - py_artifact_enabled | bool
 
+    - name: Backup the original repo-build playbook
```

Comment:
Instead of doing this copy/restore mechanism, why not just reset the git repo to the original state at the appropriate SHA?

Reply:
There may be other modifications at play and I'd rather not deal with unwinding them. At some point I hope to find a better way to accomplish that playbook without running any of those tasks.

Reply:
The plan has always been to push the staging functionality upstream, as described in https://github.com/openstack/openstack-ansible-specs/blob/master/specs/queens/deployment-stages.rst and https://github.com/openstack/openstack-ansible-specs/blob/master/specs/queens/python-build-install-simplification.rst so its time here is limited.